ogr/services/github.py

```diff
@@ -102,6 +106,13 @@ def __init__(
             )
         self.read_only = read_only
 
+    def __str__(self) -> str:
+        return (
+            f"Github-Project("
+            f"namespace= \"{self.namespace}\""
+            f"repo: \"{self.repo}\")"
```

Comment:
You've forgot to  repo parameter..;-)

```suggestion
            f"repo:=\"{self.repo}\")"
```

Reply:
My bad :(. I will make the changes. Thanks for your valuable feedback !